compiler+runtime/include/cpp/jank/runtime/obj/repeat.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <jank/runtime/behavior/seqable.hpp>
+
+namespace jank::runtime
+{
+  template <>
+  struct static_object<object_type::repeat> : gc
+  {
+    static constexpr native_bool pointer_free{ false };
+    static constexpr native_integer INFINITE{ -1 };
+    using bounds_check_t = native_bool (*)(object_ptr);
```

Comment:
This is unused, as far as I can tell.

Reply:
Removed `bounds_check_t`, the other two are used.